python/pyspark/sql/catalog.py

```diff
@@ -133,6 +133,38 @@ def listColumns(self, tableName, dbName=None):
                 isBucket=jcolumn.isBucket()))
         return columns
 
+    def tableExists(self, tableName, dbName=None):
```

Comment:
Let's add this into docs since https://github.com/apache/spark/pull/33392 is merged.

Reply:
Done